notification_configuration.go

```diff
@@ -93,12 +93,12 @@ type NotificationConfiguration struct {
 
 // DeliveryResponse represents a notification configuration delivery response.
 type DeliveryResponse struct {
-	Body       string      `json:"body"`
-	Code       int         `json:"code"`
-	Headers    http.Header `json:"headers"`
-	SentAt     time.Time   `json:"sent-at,iso8601"`
-	Successful bool        `json:"successful"`
-	URL        string      `json:"url"`
+	Body       string      `jsonapi:"attr,body"`
```

Comment:
This isn't going to work. I'm actually in the process of fixing this. See this slack thread: https://hashicorp.slack.com/archives/C01BR9FR8KZ/p1615941703005300

Reply:
gotcha. I will remove the `iso8601` annotation though since the json parser doesn't understand it. We'll just have to remember to put it back if jsonapi is ever supported.